docs/INSTALL.md

```diff
@@ -207,8 +207,8 @@ prefixed by `litecoind`.
 To configure your bitcoind backend for use with lnd, first complete and verify
 the following:
 
-- The `bitcoind` instance must be configured with `--txindex` just like `btcd`
-  above
+- The `bitcoind` instance can optionally be configured with `--txindex` just like `btcd`
+  above.
```

Comment:
I'd just remove this line like it's done in: https://github.com/lightningnetwork/lnd/pull/1148/files

Reply:
Agreed, going with #1148